internal/runners/clean/run.go

```diff
@@ -3,33 +3,63 @@ package clean
 import (
 	"os"
 
+	"github.com/ActiveState/cli/internal/errs"
+	"github.com/ActiveState/cli/internal/fileutils"
 	"github.com/ActiveState/cli/internal/locale"
+	"github.com/ActiveState/cli/internal/logging"
+	"github.com/ActiveState/cli/internal/runners/prepare"
 )
 
 func (u *Uninstall) runUninstall() error {
 	err := removeCache(u.cfg.CachePath())
 	if err != nil {
-		return err
+		u.reportError(locale.Tl("uninstall_remove_cache_err", "Failed to remove cache directory."), err)
```

Comment:
This breaks our regular error reporting, which has UX styling. It'd be more appropriate to combine these errors and return them at the end.